Introduce PATCH request into webservice API (#1386)
PrestaShop/docs: webservice/getting-started.md

{{% notice note %}}
A response obtained with "display" other than "full" can't be used in a **PUT** (update) request, because the `WebserviceRequest` class validation for fields is the same for **POST** (create) and **PUT** (update).

> **PATCH** method allow you to use a partial display !

Reviewer: ```suggestion
The **PATCH** method allows using a partial display.
```

Also, doesn't this statement cancel the previous one (just above)? Then we should remove it too, right ?

Reviewer 2: @PrestaEdit you forgot about this one

Author: @kpodemski No, I was just away from keyboard between the two suggestions changes ;)

To edit an existing resource: **GET** the full XML file for the resource you want to change (example `/api/addresses/1`), edit its content as needed, then send a **PUT HTTP request** with the whole XML file as a body content to the same URL again.

### Partially update a resource

> To partially edit an existing resource: **GET** a part of the XML file for the resource you want to change (example `/api/addresses/1`), edit its content as needed, then send a **PATCH HTTP request** with the partial XML file as a body content to the same URL again.

Reviewer: ```suggestion
To partially edit an existing resource: **GET** a part of the XML file for the resource you want to change (example `/api/addresses/1`), edit its content as needed, then send a **PATCH HTTP request** with the partial XML file as the body content to the same URL again.
```
